include/upipe/ubuf_pic.h

```diff
@@ -388,7 +397,69 @@ static inline int ubuf_pic_blit(struct ubuf *dest, struct ubuf *src,
         int plane_vsize = extract_vsize / src_vsub;
 
         for (int i = 0; i < plane_vsize; i++) {
-            memcpy(dest_buffer, src_buffer, plane_hsize);
+            if (!alpha_plane || threshold == 0) {
+                memcpy(dest_buffer, src_buffer, plane_hsize);
+            } else if (threshold != 0xff) {
+                for (int j = 0; j < plane_hsize; j += 16) {
```

Comment:
handle subpicture expiration: I'd prefer to keep the old behaviour (subpics are blitted until they are replaced) because some people rely on it (upipe_blit is used for other things than subtitles). You could instead have a pipe in front of upipe_blit that checks duration and emits an empty uref when the subtitle is over.